src/components/Waivers/WaiversList.jsx

```diff
@@ -28,6 +28,12 @@ function WaiversList({
 
   const league = useSelector(selectLeague);
 
+  useEffect(() => {
+    axios.get(`/stock/bank/${user.id}/${league}`)
```

Comment:
remove this from useEffect() and create a new method outside of the hook. Then call that function inside useEffect